### DISCUSSION TITLE
trying to live video stream using Fastapi

### DISCUSSION THREAD

**Discussion by pradeepkc11** [fastapi/fastapi#11277] [Questions]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

while i am trying to build Fastapi with html to live stream video but i am getting this error "no video with supported format and MIME type found" can any please help me with this error

**Comment by StephennFernandes** [comment]
@pradeepkc11 hey is there any update on this ?

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
